configure

```diff
@@ -683,7 +683,7 @@ if [ "$TF_NEED_OPENCL" == "1" ]; then
 while true; do
   fromuser=""
   if [ -z "$HOST_CXX_COMPILER" ]; then
-    default_cxx_host_compiler=$(which clang++-3.6 || true)
+    default_cxx_host_compiler=$(which g++-4.8 || true)
```

Comment:
We locked to GCC-4.8 as it was default installation on Ubuntu 14.04.
This has been changed in 50197b9792e3eb262201ee143cc82546a151e388 and d777320c6a9b730449f744a5ad8d068da12953ed
However, on ubuntu 16.04 user will have to add "-D_GLIBCXX_USE_CXX11_ABI=0" manually to the command line if there is custom compiler installed on the machine.